js/apps/system/_admin/aardvark/APP/react/src/views/collections/indices/addIndex/IndexFieldsHelper.ts

```diff
@@ -0,0 +1,33 @@
+import * as Yup from "yup";
+
+export const commonFieldsMap = {
+  fields: {
+    label: "Fields",
+    name: "fields",
+    type: "text",
+    isRequired: true,
+    tooltip: "A comma-separated list of attribute paths.",
+    initialValue: ""
+  },
+  name: {
+    label: "Name",
+    name: "name",
+    type: "text",
+    tooltip:
+      "Index name. If left blank, a name will be auto-generated. Example: byValue",
+    initialValue: ""
+  },
+  inBackground: {
+    label: "Create in background",
+    name: "inBackground",
+    type: "boolean",
+    initialValue: true,
```

Comment:
Is `initialValue` `string` or `boolean` or both? Need some context for this.

Reply:
for boolean type fields, it will be boolean. So it can be both